apps/studio/components/interfaces/App/CommandMenu/OrgProjectSwitcher.tsx

```diff
@@ -0,0 +1,104 @@
+import { Forward, Wrench, Building } from 'lucide-react'
+import { useMemo } from 'react'
+
+import { useProjectsQuery } from 'data/projects/projects-query'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { PageType, useRegisterCommands, useRegisterPage, useSetPage } from 'ui-patterns/CommandMenu'
+import { IS_PLATFORM } from 'common'
+import { COMMAND_MENU_SECTIONS } from './CommandMenu.utils'
+
+const PROJECT_SWITCHER_PAGE_NAME = 'Switch project'
+const ORGANIZATION_SWITCHER_PAGE_NAME = 'Switch organization'
+
+export function useProjectSwitchCommand() {
```

Comment:
This hook is just copied from the original ProjectSwitcher.tsx before the rename